cypress/integration/projectSettings/project_settings.ts

```diff
@@ -233,7 +233,7 @@ describe("Repo Settings", () => {
     it("Reorders the commands", () => {
       cy.dataCy("array-down-button").click();
 
-      cy.dataCy("save-settings-button").click();
+      cy.dataCy("save-settings-button").click({ force: true });
```

Comment:
I think it might work if you just do ` cy.dataCy("save-settings-button").scrollIntoView().click()` here 

(I tested on this [patch](https://spruce.mongodb.com/version/62d1b0da3066155eda46f370/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC))